logstash-core/lib/logstash/api/lib/app/service.rb

```diff
@@ -40,4 +44,15 @@ def get(key)
     end
     LogStash::Json.dump(data)
   end
+
+  private
+
+  def has_counters?
```

Comment:
well, it collects metrics somehow, as it collect the periodic poller metrics anyway. this is only a testing issue for now.